### ISSUE TITLE
Barrels and Tanks, Eventual better mixing, and having seperation, division of reagents.

### ISSUE THREAD

**Issue by GreenboxGames** [naelstrof/KoboldKare#120]
Mobile barrels and/or Tanks a lot like the one in the cumflator that are mobile would be nice. I say and or because tanks would be larger and have capacities for Barrels in the 100-200 range and tanks being 250+. Maybe having the ejector nozzle on top that you activate to eject the contents.

I find myself really wanting better means of organization and manipulation of reagents. It gets pretty fiddly in a non-fun way IMO. Mixing is basically the only thing we have and could use a later game machine or method to separate the contents of a mixed product. Perhaps as easy as putting the item in the already existing squeezer instead of a kobold and activating it would push out the first reagent on the stack. Also having a means of cutting a product into two halves might be nice.

**Comment by naelstrof** [comment]
Yeah, in fact I originally planned to make machines that you described, and I fully intended for alchemy to be extremely ghetto so that the machines would be necessary for accurate mixing. 

I ended up cutting the content because there isn't very many reagents. Would be fun to add them back as reagents get more flushed out.

**Comment by GreenboxGames** [comment]
Makes sense. I feel more and more as I play and look at the prefabs that I'm starting with bigger ideas and should maybe think about some of the smaller mechanical aspects that need fleshing out. I'm an old and it's kinda interesting after all these years to find a project I'm this interested and excited about.
